src/java.management/share/classes/javax/management/MXBean.java

```diff
@@ -784,7 +786,13 @@ another proxy (like {@code ModuleMXBean.getProduct()} which
     then the item in the {@code CompositeType} is called {@code name}
     and has type {@code SimpleType.BOOLEAN}.
 
-    <p>Notice that the first character (or code point) is converted to
+    <p>If the class is a {@link Record} and the getter is a component
+       accessor for a record component {@code name} of type <em>T</em>,
+       then the item in the {@code CompositeType} has the same name
+       than the record component, and has type <em>opentype(T)</em>.</p>
```

Comment:
The update to the MXBean spec looks good. Do you have a typo here, I assume it should say "as the record component".

Reply:
Right thanks. Qualifying it as "a typo" was too kind ;-)
I have updated the code. I also moved the `record` example after the `from` example to respect the order of precedence.